config/migrations/2025/20250121103936-extended-sync-op/20250121104059-flush-graphs.sparql

```diff
@@ -0,0 +1,5 @@
+CLEAR GRAPH <http://mu.semte.ch/graphs/landing-zone/op-public>
```

Comment:
This file can be deleted, it's the first time we have an op sync so no previous landing zone (it's where the consumed triples go at first) and no remapping graph (the graph where the mapping rules are applied)